front/src/Adverts.js

```diff
@@ -89,7 +89,11 @@ const AdvBody = props => {
         <div className="grid-container">
           <div className="item1">{row.title}</div>
           <div className="item2">
-            <img src={userphoto} alt="ph" width="180" height="180" />
+            {row.photo_url = = null ? (
+              <img src={userphoto} alt="ph" width="180" height="180" />
+            ) : (
+              <img src={row.photo_url} alt="ph" width="180" height="180" />
+            )}
```

Comment:
Small detail - if we are only determining the src for the photo, and the whole img tag is the same, maybe we can only change the img url variable in ternery operator. E.g. 
{let source = row.photo.url == null ? userphoto : row.photo_url}             
 <img src={source} alt="ph" width="180" height="180" />